From 7c6837f27e22742fee2f32ec2680e6f6492bceb2 Mon Sep 17 00:00:00 2001
From: Chen Meng J <meng.j.chen@intel.com>
Date: Thu, 13 Sep 2018 11:05:54 +0800
Subject: [PATCH 49/54] media: intel-ipu4: ox03a10: horizontal mirror

Message for Open Source:
horizontally mirror the image.

Message for Internal:
[Feature] sensor ox03a10 driver enabling.
horizontally mirror the image.

Change-Id: I250c853ba0fe9a2ea5905ca09b029512a199fc7b
Tracked-On: #H1407872200
Signed-off-by: Chen Meng J <meng.j.chen@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h b/drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h
index 47bc9c6..43bde8c 100644
--- a/drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ox03a10_configuration.h
@@ -247,11 +247,11 @@ struct crl_subdev_rect_rep ox03a10_1920_1088_rects[] = {
 	{ 0x380e, CRL_REG_LEN_08BIT, 0x05 },
 	{ 0x380f, CRL_REG_LEN_08BIT, 0x40 },
 	{ 0x3810, CRL_REG_LEN_08BIT, 0x00 },
-	{ 0x3811, CRL_REG_LEN_08BIT, 0x08 },
+	{ 0x3811, CRL_REG_LEN_08BIT, 0x09 },
 	{ 0x3813, CRL_REG_LEN_08BIT, 0x04 },
 	{ 0x381c, CRL_REG_LEN_08BIT, 0x08 },
 	{ 0x3820, CRL_REG_LEN_08BIT, 0x00 },
-	{ 0x3821, CRL_REG_LEN_08BIT, 0x20 },
+	{ 0x3821, CRL_REG_LEN_08BIT, 0x24 },
 	{ 0x3822, CRL_REG_LEN_08BIT, 0x14 },
 	{ 0x3832, CRL_REG_LEN_08BIT, 0x00 },
 	{ 0x3833, CRL_REG_LEN_08BIT, 0x01 },
@@ -703,11 +703,11 @@ struct crl_subdev_rect_rep ox03a10_1920_1088_rects[] = {
 	{ 0x380e, CRL_REG_LEN_08BIT, 0x05 },
 	{ 0x380f, CRL_REG_LEN_08BIT, 0x40 },
 	{ 0x3810, CRL_REG_LEN_08BIT, 0x00 },
-	{ 0x3811, CRL_REG_LEN_08BIT, 0x08 },
+	{ 0x3811, CRL_REG_LEN_08BIT, 0x09 },
 	{ 0x3813, CRL_REG_LEN_08BIT, 0x04 },
 	{ 0x381c, CRL_REG_LEN_08BIT, 0x08 },
 	{ 0x3820, CRL_REG_LEN_08BIT, 0x00 },
-	{ 0x3821, CRL_REG_LEN_08BIT, 0x20 },
+	{ 0x3821, CRL_REG_LEN_08BIT, 0x24 },
 	{ 0x3822, CRL_REG_LEN_08BIT, 0x14 },
 	{ 0x3832, CRL_REG_LEN_08BIT, 0x00 },
 	{ 0x3833, CRL_REG_LEN_08BIT, 0x01 },
-- 
1.9.1

